using-timescaledb/telemetry.md

```diff
@@ -1,5 +1,7 @@
 # Telemetry and Version Checking
-Starting in v0.12.0 we enable anonymous usage sharing to help us better understand and assist Timescale users. We emphasize that privacy of our users is paramount, so we do not collect any identifying information. The following is an example of the JSON that is sent to our servers about a specific deployment:
+Starting in v0.12.0 we enable anonymous usage sharing to help us better understand and assist Timescale users, as
```

Comment:
We might need to change this to a different version depending on when and what we release